src/autocomplete/index.js

```diff
@@ -140,23 +169,25 @@ class AutoCompleteClab {
   }
 
   _handleArrows(type) {
-    let HIdx = getIndex(this._currentHint, this.results);
+    const HIdx = getIndex(this._currentHint, this.results);
     let item;
     switch(type) {
       case 'up':
         item = this.results[HIdx - 1];
-        if(typeof item == 'object') {
+        if(typeof item === 'object') {
```

Comment:
@Arual90 `typof item === 'object'` it's a bit too generic... Can we do a better check?